tests/payloads/cocoa.json

```diff
@@ -576,7 +576,7 @@
         "name": "/Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/Library/CoreSimulator/Profiles/Runtimes/iOS.simruntime/Contents/Resources/RuntimeRoot/usr/lib/system/libsystem_containermanager.dylib",
         "revision_version": 0,
         "type": "apple",
-        "uuid": "F32F9759-4AD2-3227-8A71-1B9938180292"
+        "uuid": "FloatF9759-4AD2-3227-8A71-1B9938180292"
```

Comment:
Invalid search/replace. Applies to all test files.

Reply:
Oh shit I wanted to undo that